packages/sanity/src/structure/comments/src/components/list/CommentsListItem.tsx

```diff
@@ -93,8 +93,20 @@ interface CommentsListItemProps {
   parentComment: CommentDocument
   readOnly?: boolean
   replies: CommentDocument[] | undefined
+  avatarConfig?: {
+    parentCommentAvatar: boolean
+    threadCommentsAvatar: boolean
+    replyAvatar: boolean
+    avatarSize: AvatarSize
+  }
 }
 
+const DEFAULT_AVATAR_CONFIG: CommentsListItemProps['avatarConfig'] = {
```

Comment:
Nitpick: I think it would be a bit tidier to move this constant along with the other constants at the top of the file